### ISSUE TITLE
Respectful code cleanup

### ISSUE THREAD

**Issue by harro** [google/ipaddr-py#126]
Pull request: https://github.com/google/ipaddr-py/pull/125

**Comment by pmoody-** [comment]
`Allow list the characters` doesn't totally make sense

**Comment by pmoody-** [comment]
that is to say, FWIW (i've had very little involvement with ipaddr for a long time), i'm happy with more inclusive language, but please make sure it makes sense.

**Comment by harro** [comment]
That is the recommended term internally. Happy to go with something else if
you prefer?

**Comment by harro** [comment]
Would 'Permit list' do?

**Comment by pmoody-** [comment]
sorry, I realized after the last update that this was the old google version. the ipaddress library is in the python stdlib now. 

https://github.com/google/ipaddr-py/blob/master/README

there haven't been any commits on this repo in 4 1/2 years. the python folks are the right place for this PR.

**Comment by harro** [comment]
Yep, it's just a cosmetic change to a couple of comments in legacy code.

I haven't been involved in the development of the library for some time, and was not involved in the replacement that is in stdlib. So I'll limit myself to proposing changes here only. Feel free to close the issue if you are happy with the status quo.

**Comment by pmoody-** [comment]
i've stamped it, i'm not sure that I can merge it though. I've not worked at google in many years and I don't think I'm part of the google organization.
